Fix api affected by changes in the org-mode

niku opened the pull request:
See also
http://orgmode.org/cgit.cgi/org-mode.git/commit/?id=0d000f568011287de71e4d1ade9769bbe0f31db0

victorolinasc commented:
Hi @niku!

Thanks for this, but it seems like it is backwards incompatible. I tried to run this on my current Emacs installation and it can't find org-babel--get-vars. I am using a Emacs 24.4.

Can we make it backwards compatible?

niku committed:
Fix api affected by changes in the org-mode

See also
http://orgmode.org/cgit.cgi/org-mode.git/commit/?id=0d000f568011287de71e4d1ade9769bbe0f31db0

niku commented:
Hi @victorolinasc !
Thank you for mention to backwards compatibility.

I fixed backwards compatibility and push again.
Would you check it?

victorolinasc commented:
Right on! Works flawlessly for me now!

Once again, thanks :+1:
